src/Sylius/Component/Translation/Factory/TranslatableFactoryInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Translation\Factory;
+
+use Sylius\Component\Resource\Factory\FactoryInterface;
+
+/**
+ * Interface for factories creating translatable resources.
+ *
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+interface TranslatableFactoryInterface extends FactoryInterface
```

Comment:
Yeah, but with Translatable, two different locales is still single resource but with 2 *Translations inside. :) We will need to see how it works out. Something for later. :) Out of scope for this PR. :dancer: